### PR TITLE
Prerelease: v1.0.0-b2

### LINKED ISSUE #845: Update RPC documentation to reflect separation from core
The rpc server is being [separated from dbt-core](https://github.com/dbt-labs/dbt/issues/3675), and now lives in the [dbt-rpc](https://github.com/dbt-labs/dbt-rpc) repo. In order for users to continue using it they will have to install it from the new repository.

This rpc server is soon to be deprecated / rewritten and a warning to this effect should be reflected in the documentation.

### LINKED ISSUE #879: Remove docs on BigQuery ingestion-time partitioned tables
This page has been unlisted for some time: https://docs.getdbt.com/docs/guides/database-specific-guides/creating-date-partitioned-tables

In `dbt-bigquery==1.0.0`, we're officially deprecating support for this functionality.

For more details, see: https://github.com/dbt-labs/dbt-bigquery/issues/20, https://github.com/dbt-labs/dbt-bigquery/pull/42

### LINKED ISSUE #880: Update docs to reflect generic tests in test-paths
Generic tests can (as of dbt-core v1.0.0) be nested in a `generic` folder under `test-paths`.  Update docs to reflect this.   Related PR:  [#4052](https://github.com/dbt-labs/dbt-core/pull/4052)

### LINKED ISSUE #881: Document new `init`
Now that https://github.com/dbt-labs/dbt-core/pull/3625 is merged...

Needs update:
- https://docs.getdbt.com/reference/commands/init
- https://docs.getdbt.com/docs/contributing/building-a-new-adapter

Areas to hit:
- What happens if you're in a preexisting dbt project
- What happens if you're not
- For project + adapter maintainers: `profile_template.yml`

### PR DESCRIPTION
**Pull request by jtcohen6**
resolves #845
resolves #879
resolves #880
resolves #881

I'll also take another look through the `b2` changelog (for `dbt-core` + plugins) to see if there's anything I missed.

### CODE REVIEW

File: website/docs/docs/contributing/building-a-new-adapter.md [hunk 1]
```diff
@@ -83,9 +83,10 @@ class MyAdapterCredentials(Credentials):
...
-
-You may also want to define an `ALIASES` mapping on your Credentials class to include any config names you want users to be able to use in place of 'database' or 'schema'. For example if everyone using the MyAdapter database calls their databases "collections", you might do:
+There are a few things you can do to make it easier for users when connecting to your database:
+- Be sure to implement the Credentials' `_connection_keys` method shown above. This method will return the keys that should be displayed in the output of the `dbt debug` command. As a general rule, it's good to return all the arguments used in connecting to the actual database except the password (even optional arguments).
+- Create a `profile_template.yml` that includes to prompt a brand-new user setting up a connection profile via the [`dbt init` command](init). See more details [below](#other-files).
```
Comment (NiallRees) [inline website/docs/docs/contributing/building-a-new-adapter.md]:
```suggestion
- Create a `profile_template.yml` to enable configuration prompts for a brand-new user setting up a connection profile via the [`dbt init` command](init). See more details [below](#other-files).
```

File: website/docs/reference/commands/init.md [hunk 2]
```diff
@@ -3,22 +3,66 @@ title: "init"
...
+
+## Existing project
+
+If you've just cloned or downloaded an existing dbt project, `dbt init` can still help you set up your connection profile so that you can start working quickly. It will prompt you for connection information, as above, and add a profile (using the `profile` name from the project) to your local `profiles.yml`, or create the file if it doesn't already exist.
+
+## profile_template.yml
+
+`dbt init` knows how to prompt for connection information by looking for a file named `profile_template.yml`. It will look for this file in two places:
+
+- **Adapter plugin:** What's the bare minumum Postgres profile? What's the type of each field, what are its defaults? This information is stored in a file called [`dbt/include/postgres/profile_template.yml`](https://github.com/dbt-labs/dbt-core/blob/main/plugins/postgres/dbt/include/postgres/profile_template.yml). If you're the maintainer of an adapter plugin maintainer, we highly recommend that you add a `profile_template.yml` to your plugin, too. See more details in [building-a-new-adapter](building-a-new-adapter).
```
Comment (NiallRees) [inline website/docs/reference/commands/init.md]:
```suggestion
- **Adapter plugin:** What's the bare minumum Postgres profile? What's the type of each field, what are its defaults? This information is stored in a file called [`dbt/include/postgres/profile_template.yml`](https://github.com/dbt-labs/dbt-core/blob/main/plugins/postgres/dbt/include/postgres/profile_template.yml). If you're the maintainer of an adapter plugin, we highly recommend that you add a `profile_template.yml` to your plugin, too. See more details in [building-a-new-adapter](building-a-new-adapter).
```

**Review by NiallRees — APPROVED** [review]
dbt init parts look great.

**Comment by jtcohen6** [comment]
Thanks for the review @NiallRees! The `init` pieces of this were the most substantive, so I'm going to go ahead with merging once the checks pass. I'd rather have this in `next` sooner than later.

**Pull request merged**